tests/plugins/test_rss_plugin.py

```
@@ -135,6 +143,40 @@ def test_rss_html_entities_unescaped(monkeypatch, plugin_config, device_config_d
     assert "&amp;" not in items[0]["title"]
 
 
+def test_rss_html_tags_stripped(monkeypatch, plugin_config, device_config_dev):
```

Ruff (0.15.7)

[warning] Unused function argument: `monkeypatch`

(ARG001)